src/main/java/trello/api/rest/RestClientAPI.java

```diff
@@ -12,57 +12,106 @@
 
 package trello.api.rest;
 
-import core.selenium.util.JsonConverter;
-import core.selenium.util.ReadJsonFile;
-import io.restassured.RestAssured;
-import io.restassured.builder.RequestSpecBuilder;
+import com.google.gson.Gson;
+import io.restassured.response.Response;
 import io.restassured.specification.RequestSpecification;
-import trello.entities.User;
-import trello.keys.NamePages;
+
+import java.util.Map;
+
+import static io.restassured.RestAssured.given;
+import static io.restassured.http.ContentType.JSON;
 
 /**
- * RestClientAPI class.
+ * Authentication class.
  *
- * @author Josue Rodriguez.
- * @version 0.0.1
+ * @author Josue Rodriguez Garcia.
+ * @version 0.0.1.
  */
-public final class RestClientAPI {
+public class RestClientAPI {
+    private RequestSpecification requestSpecification;
+
+    /**
+     * Constructor method.
```

Comment:
Please add a white space bellow this line.

Reply:
Solved